### ISSUE TITLE
RichTextLabel doesn't update scroll if scroll_active is false

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#39211]
**Godot version:**

3.2.2 beta3

**Issue description:**

I have a RichTextLabel with `scroll_active` set to false, but I am manually scrolling it. Here's my magical code that keeps the proper number of lines displayed (magical, because I was surprised it works xd):
```
if contents.get_visible_line_count() > 4:
	contents.get_v_scroll().value += 3
```
I am incrementing my `visible_characters` and scroll the label if necessary. This works ok until the text ends. I'm only incrementing the `visible_characters` until the text is fully displayed. If the scroll is fast enough before the text ends, everything is fine.
![cOWt4DcU3B](https://user-images.githubusercontent.com/2223172/83365362-c187f480-a3a7-11ea-8602-9aa2f87b2fc0.gif)
However when the text ends before the scroll ends, I get this:
![mMmddZ2CGv](https://user-images.githubusercontent.com/2223172/83365366-cba9f300-a3a7-11ea-8d5a-87df98d5c40f.gif)
The label is stuck halfway in the line. And my problem is that in both cases, the final scroll value is the same. Which means that when you scroll RichTextLabel normally, the label doesn't get scrolled visually (it's not updated). It only updates when you change `visible_characters`, which causes redraw and displays proper scroll.

The workaround is to keep incrementing `visible_characters`, but IMO the scroll should just update the label even if it's "inactive", otherwise it's inconsistent.

**Minimal reproduction project:**

[ReproductionProject.zip](https://github.com/godotengine/godot/files/4708679/ReproductionProject.zip)
Compare results when holding Space and when not.

**Comment by dalexeev** [comment]
Duplicate of #37720?

**Comment by KoBeWi** [comment]
No, it's different issue.

**Comment by dreamsComeTrue** [comment]
So maybe check that one PR I made back then? 
https://github.com/godotengine/godot/pull/36366

**Comment by KoBeWi** [comment]
@dreamsComeTrue Yeah, it fixes this issue.

**Comment by KoBeWi** [comment]
Fixed by #36366
